packages/conf-zmq/conf-zmq.0.1/opam

```diff
@@ -2,7 +2,9 @@ opam-version: "1"
 maintainer: "francois.berenger@inria.fr"
 homepage: "http://zeromq.org/"
 license: "LGPL"
-build: [["cc" "test.c" "-lzmq"]]
+build: [
+  ["sh" "-c" "cc test.c -I/usr/local/include -L/usr/local/lib -lzmq"]
+]
```

Comment:
Why the explicit call to `sh` rather than something like the following?

```
["cc" "test.c" "-I/usr/local/include" "-L/usr/local/lib" "-lzmq"]
```